### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by rocketninja7**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -196,8 +202,8 @@ _Details coming soon ..._
 | **delete**       | `delete MODULE_CODE` <br> e.g., `delete CS2040S`                                    |
 | **edit**         | `edit MODULE_CODE [y/YEAR] [s/SEMESTER] [g/GRADE]` <br> e.g., `edit MA2001 y/1 s/2` |
 | **info**         | `info MODULE_CODE`<br> e.g., `info CS3230`                                          |
-| **CalculateCap** | `CalculateCap`                                                                      | 
-| **CalculateMC**  | `CalculateMC`                                                                       |
+| **calculateCAP** | `calculateCap`                                                                      |
```
Comment (marquestye) [inline docs/UserGuide.md:205]: "calculateCap" results in "Unknown command" being displayed to the user. The correct format is "calculateCAP".

**Review by marquestye — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (rocketninja7) [reply docs/UserGuide.md]: Fixed

**Review by xCOLOURx — APPROVED** [review]
LGTM 👍

**Comment by rocketninja7** [comment]
#122

**Review by xCOLOURx — APPROVED** [review]
LGTM 👍

**Pull request merged**